src/app/app.module.ts

```diff
@@ -80,7 +95,18 @@ import { AppAndPageTitleStrategy } from './title-strategy';
   ],
   bootstrap: [AppComponent],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(
+    entityDefinitionService: EntityDefinitionService,
+    entityDataService: EntityDataService,
+    ssarDataService: SelfSubjectAccessReviewDataService,
+    organizationDataService: OrganizationDataService
+  ) {
+    entityDefinitionService.registerMetadataMap(entityMetadataMap);
+    entityDataService.registerService(selfSubjectAccessReviewEntityKey, ssarDataService);
```

Comment:
```suggestion
    entityDataService.registerService(selfSubjectAccessReviewEntityKey, inject(SelfSubjectAccessReviewDataService));
```

Maybe you can do this like this. So, you don't need so many arguments in the constructor.